projects/storefrontlib/src/cms-components/user/register/register.component.spec.ts

```diff
@@ -424,6 +426,7 @@ describe('RegisterComponent', () => {
   describe('onRegisterUserSuccess', () => {
     beforeEach(() => {
       spyOn(anonymousConsentService, 'giveConsent').and.stub();
+      spyOn(anonymousConsentService, 'withdrawConsent').and.stub();
```

Comment:
You can remove the testing from the consent in here. It is tested at the bottom and this logic was removed from the `onRegisterUserSuccess` function